src/ao3/works.py

```diff
@@ -263,6 +266,73 @@ def bookmarks(self):
         # bookmarked this, but for now just return the number.
         return int(self._lookup_stat('bookmarks').contents[0])
 
+    @property
+    def bookmark_users(self):
```

Comment:
Since this method is making network requests, it shouldn’t really be a property, it should be a standard method. It’s not great to have properties that might throw (although I realise this code is not especially consistent on this point!)